tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
I think that if `lib` is not specified in tsconfig.json, then `dom` is part of the default libs brought in per https://www.typescriptlang.org/docs/handbook/compiler-options.html and DOM is necessary in this project because highlight.js is a dependency.